src/crush/CrushWrapper.h

```diff
@@ -374,9 +374,12 @@ class CrushWrapper {
   }
   int get_type_id(const std::string& name) const {
     build_rmaps();
-    if (type_rmap.count(name))
-      return type_rmap[name];
-    return -1;
+    auto found = type_rmap.find(name);
+    if (found != type_rmap.end()) {
+      return found->second;
```

Comment:
Should we tag this with the `performance` label?

Reply:
sure.